### ISSUE TITLE
[FEATURE REQUEST]

### ISSUE THREAD

**Issue by Tyloon1e** [brainsynder-Dev/SimplePets#118]
**Is your feature request related to a issue? If so please describe.**

**Describe the suggestion (Use a lot of details if it is complex)**
Hello when will SimplePets V5 be available for 1.16.5 servers? Thanks

**Screenshots/Videos (If you have any that will help us)**

**Additional Information**

**Comment by Thatsmusic99** [comment]
I made a Discord announcement regarding this on the 11th of June, 2021:

![](https://i.gyazo.com/71aa69a7987b4ffe4a781715396029e7.png)

The final v5 build to support 1.16.5 is [build 26](https://ci.pluginwiki.us/job/SimplePets_v5/26/). However, it does not work perfectly and may still have bugs.

We still have the 1.16.5 code for SimplePets on the v5.0 recode branch, so if you or another developer has the knowledge to implement all the recent features into that code, feel free to. However, I did attempt this recently and was unable to due to all the new features SP v5 relies on in 1.17.

TL:DR; it no longer supports 1.16.5 and neither I nor Brian plan on re-implementing support for the feature.
